marimo/_output/show_code.py

```diff
@@ -105,12 +108,27 @@ def factorial(n: int) -> int:
                     as_html(output),
                 ]
             )
-        else:
+        elif position == "below":
             return vstack(
                 [
                     as_html(output),
                     code_editor(value=code, disabled=True, min_height=1),
                 ]
             )
+        elif position == "left":
+            return hstack(
+                [
+                    code_editor(value=code, disabled=True, min_height=1),
+                    as_html(output),
+                ]
```

Comment:
is the default `widths` here sufficient? `widths='equal'` might be a good option if you want them split half-and-half